staging/src/k8s.io/kubectl/pkg/cmd/portforward/portforward.go

```diff
@@ -387,7 +400,14 @@ func (o PortForwardOptions) Validate() error {
 
 // RunPortForward implements all the necessary functionality for port-forward cmd.
 func (o PortForwardOptions) RunPortForward() error {
```

Comment:
TODO follow up we may indicate to people they should use the new one
```
	// Deprecated: Use RunPortForwardContext instead, which allows 
	// to cancel port forwarding.
```

Reply:
Added `Deprecation` warning to the function comments.